wsd/LOOLWSD.cpp

```diff
@@ -1791,6 +1791,20 @@ void LOOLWSD::autoSave(const std::string& docKey)
     }
 }
 
+void LOOLWSD::setLogLevelsOfKits(const std::string& level) {
```

Comment:
The C++ code in this project uses braces on new lines. Please keep the code consistent.